Examples/Swift/UserTrackingModesExample.swift

```diff
@@ -118,24 +118,37 @@ class UserLocationButton : UIButton {
     }
     
     func updateArrow(for mode: MGLUserTrackingMode) {
-        var stroke: CGColor
-        switch (mode) {
+        var arrowStrokeColor: CGColor
+        var arrowPosition: CGPoint
```

Comment:
Apparently `arrowPosition` is not being used, so I'll probably end up removing it.